advocacy_docs/partner_docs/DBeaverGuide/05-UsingDBeaver.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: 'Using DBeaver'
+description: 'Walkthroughs of example usage scenarios'
+---
+Once an instance of EDB Postgres Advanced Server or EDB Postgres Extended is connected to DBeaver, you can access the capabilities of the integration demonstrated in the next section. 
```

Comment:
We should change the order with DBeaver leading the sentence.  Below is what it should be.

Once Dbeaver is connected to an instance of EDB Postgres Advanced Server or EDB Postgres Extended , you can then access the capabilities of the integration demonstrated in the next section.